src/index.html

```diff
@@ -7,12 +8,66 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      data-qa="card"
+      class="card"
+    >
+      <img
+        src="../src/images/imac.png"
+        alt="imac"
+      />
+
+      <div class="card__t">
+        <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+      </div>
+
+      <div class="card__c">
+        <p class="card__category">Product code: 195434</p>
+      </div>
+
+      <div class="card__grade">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__review">Reviews: 5</div>
+      </div>
+
+      <div class="card__p">
+        <div class="card__price">Price:</div>
+        <div class="card__geld">$2,199</div>
```

Comment:
The class `card__geld` is not descriptive and is not in English. Class names should be in English and describe the content or function. Consider renaming it to `card__price-value` or `card__price-amount`.